superduperdb/backends/mongodb/metadata.py

```diff
@@ -176,18 +176,29 @@ def delete_component_version(
         if self._component_used(type_id, identifier, version=version):
             raise Exception('Component version already in use in other components!')
 
-        self.parent_child_mappings.delete_many(
-            {'parent': Component.make_unique_id(type_id, identifier, version)}
-        )
-
-        return self.component_collection.delete_many(
+        delete_result = self.component_collection.delete_many(
             {
                 'identifier': identifier,
                 'type_id': type_id,
                 'version': version,
             }
         )
 
+        parent_unique_id = Component.make_unique_id(type_id, identifier, version)
+
+        # TODO: Do we need to delete the child component here?
+        # We delete the child component in SQLAlchemyMetadata,
+        # but not in MongoMetaDataStore
```

Comment:
Do we need to delete the child component here, We delete the child component in SQLAlchemyMetadata,
but not in MongoMetaDataStore